crds/tests/test_config.py

```diff
@@ -23,6 +27,49 @@
 TEST_MAPPATH = os.path.join(CRDS_TESTING_CACHE, "mappings")
 TEST_TEMP_DIR = tempfile.mkdtemp(prefix='crds-test-')
 
+
+def generate_test_data():
+    import asdf
+    from roman_datamodels import datamodels as rdd
+    from roman_datamodels.testing import factories
+
+    reftype = {
+        'flat': factories.create_flat_ref,
+        'dark': factories.create_dark_ref,
+        'gain': factories.create_gain_ref,
+        'mask': factories.create_mask_ref,
+        'readnoise': factories.create_readnoise_ref
+    }
+
+    test_files = {
+        'roman_wfi16_grism_flat_small.asdf': 'flat',
```

Comment:
Can you add these three files to .gitignore?  Otherwise we'll be accidentally checking them in when submitting unrelated PRs.

Reply:
sure